content/updates/2025-09-09.md

```diff
@@ -0,0 +1,43 @@
+---
+author: ["Mariatta"]
+title: "Meeting Minutes: Sep 9, 2025"
+date: "2025-09-09T17:49:01-05:00"
+description: "Meeting Minutes from Python Docs Editorial Board: Sep 9, 2025"
+summary: "Meeting Minutes from Python Docs Editorial Board: Sep 9, 2025"
+tags: ["minutes"]
+categories: ["minutes"]
+series: ["Meeting Minutes"]
+ShowToc: false
+TocOpen: false
+members: ["Guido van Rossum", "Joanna Jablonski", "Mariatta", "Ned Batchelder"]
+---
+
+
+
+### Notes
+
+* We’re a bit disorganized in tracking action items
+* We'll go through the action items and todos from the last meeting
+* TODO: find out where they are coordinating online (mariatta)
+* Who are the Edu & Outreach? Kattni, Keith the ee, Nicholas Tollervey, Jeff Jacobson, Sheena 
+*  What is the purpose? To learn what they’re doing, what we’re doing, what’s the overlap.
+* TODO: find out from the translators what happens when the source docs change
+    * Are we still needed here?
+* There is inconsistency with the different translations groups. Do they need to be consistent?
+* We now get transifex funded through PSF core devs fund. 
+* But there are other competing options.
+* Do we want to remain in the business of translations? Can we delegate to another group?
+* Should we be explicit in saying we want a sub WG? What do we want this group to be?
+    * To coordinate and to support the translations effort
+    * Example: Adam Turner did all the work about Transifex
```

Comment:
Not all the work, I consulted a lot with some of the translation coordinators which was very valuable!